### ISSUE TITLE
Homepage missing H1 element

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#60]
All pages should have at least one h1 element.

**Comment by andrewmarino** [comment]
FIX: We can add a hidden h1 in the first page module and hide it with .screen-reader-text.
